documentation/schema.json

```diff
@@ -81,6 +81,17 @@
           "$ref": "#/definitions/ProcessFilterOptions"
         }
       ]
+    },
+    "Logging": {
```

Comment:
How does our schema compare to the "official" one? https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/appsettings.json